web_widget_color/README.rst

```diff
@@ -0,0 +1,101 @@
+Color widget for Odoo web client
+================================
+
+This module aims to add a color picker to Odoo.
+
+It's a `jsColor <http://jscolor.com/>`_ lib integration.
+
+
+Features
+========
+
+* The picker allow the user to quickly select a color on edit mode
+
+  |picker|
+
+  .. note::
+
+      Notice how html code and the background color is updating when selecting a color.
+
+
+* Display the color on form view when you are not editing it
+
+  |formview|
+
+* Display the color on list view to quickly find what's wrong!
+
+  |listview|
+
+
+Usage
+=====
+
+You need to declare a char field of at least size 7::
+
+    _columns = {
```

Comment:
It seems that we do not need 7.0 API example in README for 11.0 version of addon.

Reply:
True, it has been removed